client.go

```diff
@@ -138,11 +177,9 @@ func (c Client[Resource]) PatchById(ctx context.Context, id string, body PatchBo
 	return resource, nil
 }
 
-// List get the resources of the collection with the specified filter. It is limited by default
-// and with a max page of 200 elements (by default).
-// If you want to take more elements, use pagination
-func (c Client[Resource]) List(ctx context.Context, options Options) ([]Resource, error) {
-	req, err := c.client.NewRequestWithContext(ctx, http.MethodGet, "", nil)
+// Patch updates resources in bulk using commands in PatchBody
+func (c Client[Resource]) Patch(ctx context.Context, body PatchBody, options Options) (*Resource, error) {
```

Comment:
From [this](https://github.com/mia-platform/go-crud-service-client/pull/5#pullrequestreview-1462037191) we should call this method PatchMany

Reply:
I agree (with myself)